src/formElementPlugins/ApplicationViewWrapperNEW.tsx

```diff
@@ -149,15 +149,72 @@ const ApplicationViewWrapper: React.FC<ApplicationViewWrapperPropsNEW> = (props)
     />
   )
 
+  const changesToResponseProps: ChangesToResponseWarningProps = {
+    currentReview,
+    isChanged,
+    isValid: validationState ? (validationState.isValid as boolean) : true,
+  }
+
+  const displayResponseWarning = !!currentReview || (isChanged && validationState.isValid)
+
   return (
     <ErrorBoundary pluginCode={pluginCode}>
       <React.Suspense fallback="Loading Plugin">
-        {parametersReady && <Form.Field required={isRequired}>{PluginComponent}</Form.Field>}
+        <div
+          style={{
+            border: displayResponseWarning ? 'solid 1px' : 'transparent',
+            borderColor: isChanged ? 'blue' : !!currentReview ? 'red' : 'black',
+            padding: '5px',
+            margin: '5px',
```

Comment:
FYI -- 'px' is the default, so you can just have the number unquoted:
```
padding: 5,
margin: 5,
```
Only need quoted strings if using other units (%, rem, etc.)

Reply:
Yes. Good point. Done: 9f16d99